modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -52,34 +52,35 @@
     || $userSingleton->hasPermission('document_repository_delete')
 ) {
     if ($action == 'upload') {
-        $puser      = $_POST['user'];
-        $category   = $_POST['category'];
-        $site       = $_POST['site']       !== '' ? $_POST['site'] : null;
-        $instrument = $_POST['instrument'] !== '' ? $_POST['instrument'] : null;
-        $pscid      = $_POST['pscid']      !== '' ? $_POST['pscid'] : null;
-        $visit      = $_POST['visit']      !== '' ? $_POST['visit'] : null;
-        $comments   = $_POST['comments']   !== '' ? $_POST['commnets'] : null;
-        $version    = $_POST['version']    !== '' ? $_POST['version'] : null;
-
-        $fileSize = $_FILES["file"]["size"];
-        $fileName = $_FILES["file"]["name"];
-        $fileType = end((explode(".", $fileName)));
-
-        // __DIR__ is the document_repository ajax directory
-        // when this script is executing. Go up a level to the
-        // document_repository module directory, and use a
-        // user_uploads directory as a base for user uploads
-        $base_path = __DIR__ . "/../user_uploads/";
-        $fileBase  = $puser . "/" . $fileName;
-
-        if (!file_exists($base_path . $puser)) {
-            mkdir($base_path . $puser, 0777);
```

Comment:
A world-executable directory is probably unnecessary here. Using `700` instead.

Reply:
I think 770 is what the installer does. Maybe remove the create entirely ? Not sure i like it in the code

Reply:
Yeah that makes sense. I think the installer didn't create the directory at some point in the past.

Reply:
This has now been changed to `770`, The install script [creates a directory](https://github.com/aces/Loris/blob/minor/tools/install.sh#L158) owned by www-data and there's no implication another user would need access to this folder -- so this should work.